### PR TITLE
feat: add basic image component

### PR DESCRIPTION
**Pull request by jacobkwan**
Follows current implementation of Image in TipTap editor

specifically, looked at
1. `src/layouts/components/Editor/extensions/Image/Image.ts`
2. `src/layouts/components/Editor/components/ImageBubbleMenu/ImageHyperlinkPopover.tsx`
3. `src/layouts/components/Editor/components/ImageBubbleMenu/ResizeImagePopover.tsx`

### CODE REVIEW

**Comment by jacobkwan** [comment]
!run chromatic

File: src/components/Image/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  src,
+  alt,
+  width,
+  href,
+  openInNewTab,
+}: ImageProps) => {
+  return (
+    <div>
+      {href ? (
+        <a href={href} target={openInNewTab ? "_blank" : undefined}>
```
Comment (dcshzj) [inline src/components/Image/Image.tsx:25]: suggestion: can add `rel="noopener noreferrer nofollow"` for external links for security.

File: src/components/Image/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { HTMLAttributeAnchorTarget } from "react"
+
+export interface ImageProps {
+  src?: string
```
Comment (dcshzj) [inline src/components/Image/Image.tsx:4]: suggestion: this should be made required

Comment (jacobkwan) [reply src/components/Image/Image.tsx]: [fc32135](https://github.com/isomerpages/isomer-components/pull/7/commits/fc32135ffdc56f7e0147075b36f8164d36ebbef6)

similar to infopic, i think its good to make alt text required too

File: src/components/Image/Image.tsx (hunk 1 above)
Comment (jacobkwan) [reply src/components/Image/Image.tsx]: [72d1921](https://github.com/isomerpages/isomer-components/pull/7/commits/72d19211abe7a233909a00117f941a55718a4fd0)

great suggestion, thanks!

**Comment by jacobkwan** [comment]
!run chromatic

**Review by dcshzj — APPROVED** [review]

**Pull request merged**